conda env create gets "CondaValueError: Value error: prefix already exists:"

## Repro steps
1. set up project [as in the README](https://github.com/benrady/pythonista#setup)
2. `make test` (succeeds)
3. `touch environment.yml`
4. `make test`

## Actual behavior
failure with:
```
root@a13ba0b282ee:/myproject# make test

CondaValueError: prefix already exists: /myproject/.venv

make: *** [Makefile:35: /myproject/.venv/.deps] Error 1
```

### Expected behavior
success with:
```
Collecting package metadata (repodata.json): done
Solving environment: done
[...]
```

--- a/Makefile
+++ b/Makefile
@@ -32,7 +32,7 @@ $(CONDA): | $(PROJECT_NAME)
 environment.yml: | $(CONDA)
 
 $(DEPS): environment.yml
-	$(CONDA) env create -f environment.yml -p $(VENV)
+	$(CONDA) env create -f environment.yml -p $(VENV) --force
 	cp environment.yml $(DEPS)
 
 .PHONY: clean

